### PR TITLE
fix(kibbeh): make mobile UI support electron app

### PR DESCRIPTION
**Pull request by amitojsingh366**
- adds mobile UI support for baklava to kibbeh

@goldyydev helped me shift all that nasty logic to it's own hook

### CODE REVIEW

File: kibbeh/src/modules/layouts/MainLayout.tsx [hunk 1]
```diff
@@ -112,17 +113,35 @@ export const MainLayout: React.FC<MainLayoutProps> = ({
 
   return (
     <>
-      <div className="fixed top-0 left-0 w-full z-10">{prepend}</div>
+      <ElectronHeader />
+      <div
+        className={`fixed left-0 w-full z-50`}
```
Comment (ajmnz) [inline kibbeh/src/modules/layouts/MainLayout.tsx]:
```suggestion
        className={`fixed left-0 w-full z-10`}
```

Why did you add a higher z-index? It messes up the black thing that darkens the screen when you open the AccountOverlay. If it's an electron thing, make it electron exclusive

File: kibbeh/src/ui/mobile/AccountOverlay.tsx [hunk 2]
```diff
@@ -79,15 +79,15 @@ export const AccountOverlay: React.FC<AccountOverlyProps> = ({}) => {
...
+        className="w-screen h-screen absolute left-0 bg-black z-10 opacity-100"
         onClick={() => close()}
         style={bgStyle}
       ></a.div>
       <a.div
         className="bg-primary-800 w-full h-full rounded-t-20 relative pt-5"
         {...bind()}
         style={{
-          bottom: `calc(-100% + ${height - 100}px)`,
+          bottom: `calc(-100% + ${height - 70}px)`,
```
Comment (ajmnz) [inline kibbeh/src/ui/mobile/AccountOverlay.tsx]:
```suggestion
          bottom: `calc(-100% + ${height - 100}px)`,
```

The margin below logout was intentional. If it's an electron thing, make it electron exclusive

**Review by ajmnz — CHANGES_REQUESTED** [review]

File: kibbeh/src/modules/layouts/MainLayout.tsx (hunk 1 above)
Comment (amitojsingh366) [reply kibbeh/src/modules/layouts/MainLayout.tsx]: done

File: kibbeh/src/ui/mobile/AccountOverlay.tsx (hunk 2 above)
Comment (amitojsingh366) [reply kibbeh/src/ui/mobile/AccountOverlay.tsx]: done

File: kibbeh/src/modules/layouts/GridPanels.tsx [hunk 3]
```diff
@@ -29,7 +31,20 @@ export const MiddlePanel: React.FC<
...
+      <div
+        className={
+          !(screenType === "fullscreen" && !stickyChildren)
+            ? `flex sticky w-full flex-col z-8 bg-primary-900 pt-5`
+            : ""
+        }
+        style={
+          screenType === "fullscreen" &&
+          isElectron() &&
+          !useHostStore.getState().isLinux
```
Comment (benawad) [inline kibbeh/src/modules/layouts/GridPanels.tsx]: instead of putting this ugly check in multiple places, why don't you create a custom hook

**Review by benawad — CHANGES_REQUESTED** [review]

File: kibbeh/src/modules/layouts/GridPanels.tsx (hunk 3 above)
Comment (amitojsingh366) [reply kibbeh/src/modules/layouts/GridPanels.tsx]: oh that's a great idea

Comment (amitojsingh366) [reply kibbeh/src/modules/layouts/GridPanels.tsx]: done
